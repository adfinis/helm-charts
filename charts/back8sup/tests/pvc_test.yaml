suite: pvc tests
templates:
  - pvc.yaml
tests:
  - it: should match snapshot
    values:
      - ../ci/default-values.yaml
    set:
      persistence.enabled: true
    asserts:
      - hasDocuments:
          count: 1
      - matchSnapshot: {}
