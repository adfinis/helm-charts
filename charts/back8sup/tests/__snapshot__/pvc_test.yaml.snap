should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: back8sup
        app.kubernetes.io/version: v0.7.5
        helm.sh/chart: back8sup-0.4.0
      name: RELEASE-NAME-back8sup
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 10Gi
